packages/vm/lib/evm/eei.ts

```diff
@@ -349,14 +365,18 @@ export default class EEI {
   async _selfDestruct(toAddress: Buffer): Promise<void> {
     // only add to refund if this is the first selfdestruct for the address
     if (!this._result.selfdestruct[this._env.address.toString('hex')]) {
-      this.refundGas(new BN(this._common.param('gasPrices', 'selfdestructRefund')))
+      this.refundGas(
+        new BN(this._common.param('gasPrices', 'selfdestructRefund'))
+      )
```

Comment:
I actually liked the `printWidth` setting being extended to 100 lines (from 800 or so) on the [old format package](https://github.com/ethereumjs/ethereumjs-config/pull/33/files#diff-5bc8394070975c0969eda8f8eb989c61L3), this gives a bit more space and there was a somewhat longer discussion around this during the introduction of config until we decided upon this.

This seems to not have made it, was this intentional? (at least I am assuming during re-formattings like the above, haven't concretely checked).

Not sure, should we keep / post-introduce this, any opinions @cgewecke @ryanio or @jochem-brouwer ?

@evertonfraga in doubt and if too much process interrupting merge here anyhow, we'll get this settled out later, eventually with some follow-up PR after discussion just with the linting updates.